assets/js/components/surveys/SurveyViewTrigger.js

```diff
@@ -37,11 +41,11 @@ export default function SurveyViewTrigger( { triggerID, ttl = 0 } ) {
 
 	const { triggerSurvey } = useDispatch( CORE_USER );
 
-	useMount( () => {
+	useEffect( () => {
```

Comment:
See associated commit for details on this one - https://github.com/google/site-kit-wp/pull/6203/commits/cced86f665a7dba9fd56d46c9858f43e45050226